protostar/testing/starkware/execution_resources_summary.py

```diff
@@ -74,6 +74,7 @@ class ExecutionResourcesSummary:
     n_steps: Statistic = field(default_factory=CountStatistic)
     n_memory_holes: Statistic = field(default_factory=CountStatistic)
     builtin_name_to_count_map: Dict[str, Statistic] = field(default_factory=dict)
+    estimated_fee: float = field(default_factory=float)
```

Comment:
Is that one going to be 0 by default? I think that might be misleading

Reply:
Yeah, we probably should go with optional here

Reply:
[make estimated_fee optional](https://github.com/software-mansion/protostar/pull/1013/commits/00572a026aeb8dcf327b3ba8a327d2e80bf00bf3)